test/Microsoft.DotNet.Watcher.Tools.FunctionalTests/Scenario/ProjectToolScenario.cs

```diff
@@ -0,0 +1,178 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
```

Comment:
IIS tools don't have functional tests as yet. Do you plan on using it there? Additionally, the semantics there is pretty different - you're primarily publishing the application and watching the side effects as opposed to running the tool as standalone. It's pretty unlikely a lot of this is re-usable in that context.